app/grandchallenge/evaluation/urls.py

```diff
@@ -21,11 +22,25 @@
 app_name = "evaluation"
 
 urlpatterns = [
-    path("", EvaluationList.as_view(), name="list"),
     path("<uuid:pk>/", EvaluationDetail.as_view(), name="detail"),
     # UUID should be matched before slugs
     path("<uuid:pk>/update/", EvaluationUpdate.as_view(), name="update"),
     path("phase/create/", PhaseCreate.as_view(), name="phase-create"),
+    path("methods/<uuid:pk>/", MethodDetail.as_view(), name="method-detail"),
+    path("methods/<slug>/", MethodList.as_view(), name="method-list"),
+    path(
+        "methods/<slug>/create/", MethodCreate.as_view(), name="method-create"
+    ),
+    path("submissions/", SubmissionList.as_view(), name="submission-list"),
+    path(
+        "submissions/<uuid:pk>/",
+        SubmissionDetail.as_view(),
+        name="submission-detail",
+    ),
+    path("results/", LeaderboardRedirect.as_view()),
+    path("leaderboard/", LeaderboardRedirect.as_view()),
+    path("<slug>/", EvaluationList.as_view(), name="list"),
+    path("<slug>/admin", EvaluationAdminList.as_view(), name="admin-list"),
```

Comment:
Try to keep the URLs consistent with the rest, slug should come first if it is necessary, list views should be grouped by objects and URLs should end with "/":

```python
    path("<slug>/methods/", MethodList.as_view(), name="method-list"),
    path(
        "<slug>/methods/create/", MethodCreate.as_view(), name="method-create"
    ),
    path("<slug>/methods/<uuid:pk>/", MethodDetail.as_view(), name="method-detail"),
    path("<slug>/evaluations/", EvaluationList.as_view(), name="list"),
    path("<slug>/evaluations/admin/", EvaluationAdminList.as_view(), name="admin-list"),
    path("<slug>/submissions/", SubmissionList.as_view(), name="submission-list"),
    path(
        "<slug>/submissions/<uuid:pk>/",
        SubmissionDetail.as_view(),
        name="submission-detail",
    ),
```
   
The redirects should remain last:
  
```
    path("results/", LeaderboardRedirect.as_view()),
    path("leaderboard/", LeaderboardRedirect.as_view()),
```

Reply:
Okay. I see that you also added the `slug` to the `submission-list`, for example. I'm not sure if we want that. We only use this to give participants an overview of all their submission (i.e. phase non-specific) through a link in the Submit menu. Do we really want this to be phase specific as well? We'd then need an extra nav-tab for each phase, which would clutter the UI I think. I would keep the submission list phase unspecific.

Reply:
OK thats fine, just thought that you had changed it as the location of those URLs changed.